mimic/imimic.py

```diff
@@ -28,13 +28,37 @@ def resource_for_region(region, uri_prefix,
         """
 
 
+class IEndpointTemplate(Interface):
+    """
+    AN :obj:`IEndpointTemplate` provides an interface for the OS-KSCATALOG
+    Admin endpoint templates, defining the fields.
+    """
+    id_key = Attribute("Unique Service ID")
+    region_key = Attribute("Region the Endpoint services")
+    type_key = Attribute("Service type, example: object-store")
+    name_key = Attribute("Service Name, example: Cloud Files")
+    enabled_key = Attribute("Whether the Endpoint is globally available")
+    publicURL = Attribute("Internet Facing URL")
```

Comment:
It looks like the attributes from here downward are using camel-casing. Please change these so that they use snake casing to match the other attributes. This will affect the implementation of the `ExampleEndpoint` as well.